USER_MANUAL.md

```diff
@@ -3146,13 +3146,17 @@ unusual or complex examples. Such attributes are typically generated by the
 compiler and used internally. We list these attributes below as a reference for
 interested readers:
 
-| Name       | Type | Backend | Reference                                                                             |
-| ---------- | ---- | ------- | ------------------------------------------------------------------------------------- |
-| `assoc`    | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
-| `comm`     | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
-| `idem`     | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
-| `unit`     | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
-| `userList` | prod | all     | No reference yet                                                                      |
+| Name           | Type | Backend | Reference                                                                             |
+|----------------| ---- | ------- |---------------------------------------------------------------------------------------|
+| `assoc`        | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
+| `comm`         | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
+| `idem`         | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
+| `unit`         | prod | all     | [`assoc`, `comm`, `idem` and `unit` attributes](#assoc-comm-idem-and-unit-attributes) |
+| `userList`     | prod | all     | Identifies the desugared form of `Lst ::= List{Elm,"delim"}`                          |
+| `predicate`    | prod | all     | Specifies the sort of a predicate label                                               |
```

Comment:
Is predicate actually used anywhere? Do the backends care about it?

@theo25 ? @ana-pantilie ?

Reply:
No, I don't think so.